types/relay-compiler/language/RelayLanguagePluginInterface.d.ts

```diff
@@ -22,12 +22,14 @@ export type FormatModule = (
       | typeof RelayConcreteNode.FRAGMENT
       | typeof RelayConcreteNode.REQUEST
       | typeof RelayConcreteNode.BATCH_REQUEST
+      | "ConcreteFragment"
```

Comment:
ts was yelling at me about this one.
also future proofed stuff by adding in the 2 that are used today below

Reply:
Does this mean that in reality `typeof RelayConcreteNode.FRAGMENT` should be `"ConcreteFragment"`?

And just for my understanding, in what way was TS complaining about these?

Reply:
to be really accurate, the relay-compiler only creates 2 types now: ReaderFragment & ConcreteRequest. Batch requests got deprecated around v1.3 & tbh i don't know when fragment & request got deprecated.

Reply:
ts was complaining when it was typechecking the test file

Reply:
Ah, hmm, ok. Well, something to look into in the future, not in any way a blocker for this PR.